### PR TITLE
Ability to save cidrBlockAssociationSet

### PR DESCRIPTION
**Pull request by davidv-cuelogic**
describe_vpcs now return Ipv6CidrBlockAssociationSet and CidrBlockAssociationSet as a list of association objects. 
This PR attempts to save both the results in ipv_6_cidr_block_association_set and cidrBlockAssociationSet attributes respectively

### CODE REVIEW

File: lib/fog/aws/models/compute/vpc.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ class VPC < Fog::Model
         attribute :tenancy,          :aliases => 'instanceTenancy'
         attribute :is_default,       :aliases => 'isDefault'
         attribute :ipv_6_cidr_block_association_set,  :aliases => 'ipv6CidrBlockAssociationSet'
+        attribute :cidrBlockAssociationSet,  :aliases => 'cidrBlockAssociationSet'
```
Comment (geemus) [inline lib/fog/aws/models/compute/vpc.rb]: For consistency I think we probably want the attribute name to be underscore separated instead of camelcase?

File: lib/fog/aws/parsers/compute/describe_vpcs.rb [hunk 2]
```diff
@@ -24,22 +28,33 @@ def end_element(name)
               case name
                 when 'item'
                   @vpc['tagSet'][@tag['key']] = @tag['value']
-                  @tag = {}
```
Comment (geemus) [inline lib/fog/aws/parsers/compute/describe_vpcs.rb]: Why is this removed? Seems like it would be unrelated to the cidr block sets and might break stuff.

**Comment by lanej** [comment]
This is covered in #389
